akka-stream/src/main/resources/reference.conf

```diff
@@ -19,6 +19,14 @@ akka {
       # or full dispatcher configuration to be used by ActorMaterializer when creating Actors.
       dispatcher = "akka.actor.default-dispatcher"
 
+      # FQCN of the MailboxType. The Class of the FQCN must have a public
+      # constructor with
+      # (akka.actor.ActorSystem.Settings, com.typesafe.config.Config) parameters.
+      # defaults to the single consumber mailbox for better performance.
+      default-mailbox {
+        mailbox-type = "akka.dispatch.SingleConsumerOnlyUnboundedMailbox"
+      }
```

Comment:
I don't think this is something we should make configurable, it should be safe to switch over to the SingleConsumerOnlyUnboundedMailbox for ActorGraphInterpreter without any opt out IMO.

Reply:
But in akka typed it's configuable。

Reply:
And this will let user beable to test other mailbox implementation, such as a resizeable array based.

Reply:
Ok, fair enough.

Reply:
I think however it should just be a config setting, not represented as an attribute. Just setting the attribute for graph will not cause an async boundary and understanding the ordering of it vs async and dispatcher attributes is too confusing compared to the very small benefit of being able to set it per stream island.

Reply:
I see，that's true.